### ISSUE TITLE
Are finetuning models available ?

### ISSUE THREAD

**Issue by wbbeyourself** [MohammadrezaPourreza/DTS-SQL#1]
Dear author, I have noticed that **DTS-SQL + DeepSeek 7B**  achieves 60.31% ex accuracy in BIRD leaderboard.
I would like to ask when the fine-tuned model can be open-sourced? Thank you very much.

**Comment by MohammadrezaPourreza** [comment]
Hi, thank you so much for your interest in our work. Both schema linker and sql generator models are available in huggingface: 
Schema linker: https://huggingface.co/MrezaPRZ/DeepSchema_BIRD
SQL generator: https://huggingface.co/MrezaPRZ/DeepSQL_BIRD

**Comment by wbbeyourself** [comment]
Great, thanks a lot. I also want to try the finetuned models of Spider dataset.
In your huggingface homepage, I only see the `MrezaPRZ/Spider_full_finetuning_deepseek`. 
Please help me to identify which one is Schema linker or SQL generator for Spider.

**Comment by wbbeyourself** [comment]
> Hi, thank you so much for your interest in our work. Both schema linker and sql generator models are available in huggingface: Schema linker: https://huggingface.co/MrezaPRZ/DeepSchema_BIRD SQL generator: https://huggingface.co/MrezaPRZ/DeepSQL_BIRD

May I ask if it is possible to provide scripts for BIRD data preprocessing, model training, and inference in order to better reproduce the results?

**Comment by wbbeyourself** [comment]
Dear author, I hope this message finds you well. I wanted to express my gratitude for providing the code to train the model on the Spider dataset. The entire process went smoothly, and I found your code to be concise, easy to understand, and truly great.

If it's not too much trouble, I was wondering if you could kindly help me with a couple of questions regarding DTS-SQL:
1. I was wondering if you could clarify whether DTS-SQL uses the entire schema of the original database or only selected schema information during the SQL Generation Fine-tuning stage?
2. I was also curious if you could kindly confirm whether the two-stage process of DTS-SQL inference is as follows: First, the DeepSchema_BIRD model is called to obtain filtered_db_schemas. Second, based on the question and filtered_db_schemas, DeepSQL_BIRD is called to generate SQL?

Thank you in advance for your time and assistance. I look forward to your reply.

**Comment by MohammadrezaPourreza** [comment]
1) Since we have two models, one schema linker and the second one is the SQL generator, the SQL generator is only trained on the correct tables during the finetuning process.
2) That is totally correct
